backend/src/main/java/com/google/capstone/TextToSpeech.java

```diff
@@ -16,26 +20,32 @@
 import com.google.cloud.texttospeech.v1.TextToSpeechClient;
 import com.google.cloud.texttospeech.v1.VoiceSelectionParams;
 import com.google.protobuf.ByteString;
-import java.io.FileOutputStream;
-import java.io.OutputStream;
-
 
 
+/*
+ * Servlet takes input of object id & text and generates an audio file from the text.
+ * Audio file can be found at 'https://storage.cloud.google.com/tts-audio/object-id'
+ */
 @WebServlet(name = "TextToSpeech", value = "/api/v1/tts")
 public class TextToSpeech extends HttpServlet {
 
+  private static final String PROJECT_ID = "igunda-isangimino-nstroupe";
+  private static final String BUCKET_NAME = "tts-audio";
+
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I am wondering why we don't return the audio file url in the response. If the client just use the url in the response instead of a pre-set url, it should be more robust to change. For example, when backend server changes the bucket name, client does not need to be changed.

Reply:
Makes sense. I will change it.